python/fek.py

```diff
@@ -2,3 +2,6 @@
 
 def paer(args):
 	print(args[0])
+
+def _values():
+	return {"paer": None}
```

Comment:
We'll have to allow generic types eventually to allow for `paer` to accept any value.